### PR TITLE
Add boost.process

### PR DESCRIPTION
**Pull request by hofbi**
Closes https://github.com/bazelbuild/bazel-central-registry/issues/2993

Took the deps from https://alandefreitas.github.io/boostdep_graph/index.html

### CODE REVIEW

**Comment by bazel-io** [comment]
Hello @bazelbuild/bcr-maintainers, modules without existing maintainers (boost.process) have been updated in this PR. Please review the changes.

**Comment by meteorcloudy** [comment]
@wep21 @Vertexwahn Can you review?

**Comment by hofbi** [comment]
@meteorcloudy I also need that label for download from github

**Review by Vertexwahn — DISMISSED** [review]

**Comment by Vertexwahn** [comment]
@hofbi You can also add yourself as a maintainer if you like

File: modules/boost.process/metadata.json
Comment (hofbi) [inline modules/boost.process/metadata.json:13]:
```suggestion
        {
            "email": "julian.amann@tum.de",
            "github": "Vertexwahn",
            "name": "Julian Amann"
        },
        {
            "github": "hofbi",
            "name": "Markus Hofbauer"
        }
```

**Review by Vertexwahn — APPROVED** [review]

**Comment by hofbi** [comment]
> @hofbi You can also add yourself as a maintainer if you like

Done, but need another review

**Comment by Vertexwahn** [comment]
@meteorcloudy Ready to merge!

**Comment by wep21** [comment]
@hofbi @Vertexwahn I've added parse header feature in #3074 because this is header only library. Could you review this PR?

**Review by meteorcloudy — APPROVED** [review]

**Comment by meteorcloudy** [comment]
Oh sorry, it seems I should not merge this one. I guess https://github.com/bazelbuild/bazel-central-registry/pull/3074 has to be a `.bcr.1` version.. 😬

**Pull request merged**
